pandas/core/frame.py

```diff
@@ -4621,9 +4624,15 @@ def drop_duplicates(
         if inplace:
             (inds,) = (-duplicated)._ndarray_values.nonzero()
             new_data = self._data.take(inds)
+
+            if ignore_index:
+                new_data.axes[1] = ibase.default_index(len(inds))
```

Comment:
Is there a different between this call and the `.drop` in the other branch except for an `inplace=True` keyword? I think should handle consistently

Reply:
no difference, my bad, i initially thought this was faster. I now change all to the same for better consistency(this pr and the other pr).

thanks for your carefulness! @WillAyd